components/application-operator/before-commit.sh

```diff
@@ -57,12 +57,15 @@ fi
 # GO TEST
 ##
 echo "? go test"
-go test ./...
+go test -short -coverprofile=cover.out ./...
```

Comment:
Running test coverage on all directories also counts the generated code. It is not covered with tests, which therefore decreases overall test coverage of the component significantly. This is the case for several components.
I do not think this is desired behavior.

Reply:
Yes, we definitely do not want to count those packages, thanks for pointing that out.
For now I just enabled the coverage to have at least a number.

Teams can further improve on their respective components as they see fit by either adding unit tests or removing packages that shall be excluded.